runtime/lua/vim/treesitter/query.lua

```diff
@@ -1,17 +1,14 @@
 local api = vim.api
 local language = require('vim.treesitter.language')
 
+---Parsed query, see |vim.treesitter.query.parse()|
 ---@class Query
```

Comment:
We can also consider changing the name of the class this time, prefixing it like `vim.treesitter.query.Query`.